feat(react-native): Add push notification docs (#5344)
aws-amplify/docs: src/fragments/lib/push-notifications/common/getting-started.mdx

<BlockSwitcher>
<Block name="Amplify CLI">

> Prerequisite: [Install and configure the Amplify CLI](/cli/start/install)

<Callout warning>

> Push Notifications requires version **10.8.0+** of the Amplify CLI. You can check your current version by typing `amplify -version` and upgrade to the latest version by typing `amplify upgrade`.

Reviewer: Feel free to ignore this one, I just think it reads better.

```suggestion
Push Notifications requires version **10.8.0+** of the Amplify CLI. You can check your current version with `amplify -version` and upgrade to the latest version with `amplify upgrade`.
```

</Callout>

> To start provisioning push notifications resources in the backend, go to your project directory and execute the command:

Reviewer: ```suggestion
To start provisioning push notification resources in the backend, go to your project directory and execute the command:
```
